.github/workflows/main.yml

```diff
@@ -12,11 +12,16 @@ on:
 
 jobs:
   prerequisites:
-    uses: pulumiverse/infra/.github/workflows/provider-prerequisites.yaml@actions-workflows-v0.0.1
+    uses: pulumiverse/infra/.github/workflows/provider-prerequisites.yaml@main
```

Comment:
I just tagged latest on `main` as [actions-workflows-v0.0.4](https://github.com/pulumiverse/infra/tree/actions-workflows-v0.0.4). Please use this tag on this and subsequent lines. Given the early stage of these reusable workflows, I want to allow for breaking changes without breaking the builds of all providers using these.